number.py

```diff
@@ -1,5 +1,7 @@
 # Import the math module using import keyword.
 import math
```

Comment:
This is a Unusedimportsys
```suggestion 
```

 from our GitHub Enhacement Bot